playbooks/ci_system_info.py

```diff
@@ -0,0 +1,43 @@
+import platform
```

Comment:
You need to call this file inside a playbook, do that here in  the same pull request.

Reply:
Couldn´t we just modify the CI script to run this script ?